### ISSUE TITLE
Subcategories not accessible

### ISSUE THREAD

**Issue by apeltzer** [donwayo/ebayKleinanzeigen#26]
Hi! 

tried looking into this a bit but couldn't find a nice way to fix this. When selling a bike, I selected the correct category:

https://www.ebay-kleinanzeigen.de/p-kategorie-aendern.html#?path=210/217/herren/

However, this requires another step to select a "Subtype" in the submission field - checking for additional categories, I could find this in the source of the page so tried adding this to additional_category_options but wasn't successful.

```
"additional_category_options": {
                "fahrraeder.type_s": "city"
            },
```

Any hint? Couldn't find anything in the docs so far... other things work fine, just if I need to specify subtypes, this seems to not work

**Comment by L480** [comment]
Quick and dirty workaround: Put a `time.sleep(300)` before `driver.close()` in kleinanzeigen.py and select the category manually in the browser after the image upload is completed.

**Comment by apeltzer** [comment]
Yeah, did something similar in the end (just watching and clicking the category myself during the image upload - which works fine even without modifications) - but "not optimal" lets say ;-)
